core/crypto.mk

```diff
@@ -47,6 +47,7 @@ CFG_CRYPTO_ECC ?= y
 CFG_CRYPTO_SM2_PKE ?= y
 CFG_CRYPTO_SM2_DSA ?= y
 CFG_CRYPTO_SM2_KEP ?= y
+CFG_CRYPTO_X25519 ?= y
```

Comment:
Add `ifeq ($(CFG_CRYPTOLIB_NAME),tomcrypt)` / `endif` and also `$(call force,CFG_CRYPTO_X25519,n,not supported by mbedtls)` in `core/core.mk` (same as for `CFG_CRYPTO_RSASSA_NA1` which is also supported by LTC only).